Formula/rotz.rb

```diff
@@ -0,0 +1,22 @@
+class Rotz < Formula
+  desc "Fully cross platform dotfile manager and dev environment bootstrapper"
+  homepage "https://volllly.github.io/rotz/"
+  url "https://github.com/volllly/rotz/archive/refs/tags/v0.9.3.tar.gz"
+  sha256 "b7556bdfbee63c9def84a6ce8b12fa41d3f7ee53e2d3b3f240d35110c9ed2f1b"
```

Comment:
I think this will work here (I also bumped the version as it is now 0.9.4):

```suggestion
  version "0.9.4"
  url "https://github.com/volllly/rotz/archive/refs/tags/v#{version}.tar.gz"
  sha256 "37fce00ae83e0e24c57a476ce4cd567cfb415d8fd3de01ff8b1c20c3061d88df"
```

This will allow us to integrate it into the CI process for `rotz` when releasing with:
`brew bump-formula-pr --version <new version here> --sha256 <sha value here>` (just an example value)

Reply:
That command will work without your changes just fine.